src/components/Button/Buttons.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components'
+import {AutoRowCleanGap} from "../Row"
+import { componentName } from '../../utils'
+import {StyledButtonsArray,Base,ButtonConfirmed,ButtonError,ButtonWithLink,CarrotButton,BaseProps,ButtonPrimary} from "."
+
+
+
+<Meta title="ButtonsMDX" />
+
+# Buttons
+
+<p> All buttons below are purely styled components buttons which accept css props and <strong>Base</strong> button is base that all of them use </p>
+
+<Canvas>
+<Story name='Styled Buttons'  component={Base} >
+<AutoRowCleanGap gap={12} >
+  {StyledButtonsArray.map((Component) => (
+      <Component style={{ width: 'auto' }} key={Component.index}>
+        {componentName(Component)}
+      </Component>
+    ))}
+    </AutoRowCleanGap>
+</Story>
+</Canvas>
+
+Below are all Basic props styled buttons accept
+
+<ArgsTable of={Base} />
+
+
+export const ButtonConfirmedTemplate=(args)=><ButtonConfirmed {...args}/>
+
+## <em>Button confirmed</em> has two states depending on confirm property
+<Canvas>
+<Story name='Button Confirmed/Active' component={ButtonConfirmed} args={{
+  content: 'Confirmed true',
+  confirmed: true,
+  width: 'fit-content',
+  altDisabledStyle: true,
+}}>
+  {ButtonConfirmedTemplate.bind({})}
+</Story>
+<Story name='Button Confirmed/Inactive' component={ButtonConfirmed} args={{
+  content: 'Confirmed false',
+  confirmed: false,
+  width: 'fit-content',
+  altDisabledStyle: true,
+}}>
+  {ButtonConfirmedTemplate.bind({})}
+</Story>
+</Canvas>
+
+
+
+export const ButtonErrorTemplate=(args)=><ButtonError {...args}/>
+
+## <em>Button error</em> has two states depending on error property
+<Canvas>
+<Story name='Button Error/Active' component={ButtonError} args={{
+  error: true,
+  content: 'Error triggered',
+  width: 'fit-content',
+}}>
+{ButtonErrorTemplate.bind({})}
+</Story> 
+<Story name='Button Error/Inactive' component={ButtonError} args={{
+  error: false,
+  content: 'No Error',
+  width: 'fit-content',
+}}>
+{ButtonErrorTemplate.bind({})}
+</Story> 
+</Canvas>
+
+
+
+
+export const ButtonWithLinkTemplate=(args)=><ButtonWithLink {...args}/>
+
+export const CarrotButtonTemplate=(args)=><CarrotButton {...args}/>
+
+## <em>Button link and Carrot Button</em> share props but have different styling
+
+<br/>
+
+> ### Button With Link
+
+<Canvas>
+<Story name='Button With Link' component={ButtonWithLink} argTypes={ButtonWithLink} args={{
+  link:
+    'https://dcabtc.com/what-is-stacking-sats#:~:text=In%20short%2C%20stacking%20sats%20is,the%20smallest%20unit%20in%20Bitcoin.',
+  text: 'Stack Sats',
+  style: { width: 'fit-content' },
+}}>
+{ButtonWithLinkTemplate.bind({})}
+</Story> 
+</Canvas>
+
+
+> ### Carrot Button
+
+<Canvas>
+<Story name='Carrot Button' component={CarrotButton} args={{
+  link: 'https://carrot.eth.limo/',
+  text: 'Carrot Button Text',
+  style: { width: 'fit-content' },
+}}>
+{CarrotButtonTemplate.bind({})}
+</Story> 
+</Canvas>
+
+<ArgsTable of={ButtonWithLink} />
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
![Screenshot 2022-07-01 at 19 22 24](https://user-images.githubusercontent.com/33226956/176941554-30b64684-e3ec-4ddc-84d2-d4cda50231b8.png)
I made them side by side so that you can all check it out. Biggest difference is when you click on docs for the mdx docs is wholly custom made and its supports markdown. And for .tsx docs its auto genrated one which I haven't done anything...